### PR TITLE
It.is a.dev

### PR DESCRIPTION
**Pull request by tr1xem**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ✔ ] The file is in the `domains` folder and is in the JSON format.
- [✔ ] You have completed your website. 
- [✔ ] The website is reachable.  
- [ ✔ ] You're not using Vercel or Netlify.  
- [ ✔ ] The CNAME record doesn't contain `https://` or `/`.   Using TXT
- [  ✔] There is sufficient information at the `owner` field.  

## Website Link/Preview
Registering for Github pages like [this](https://lawliet.tk)

### CODE REVIEW

File: domains/it.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+        "owner": {
+           "username": "tr1xem",
+           "email": "kmtechnical2006@gmail.com",
+           "discord": "885063317079592961"
+        },
+    
+        "record": {
+             "CNAME": "superuser-l.github.io",
+             "TXT": [
```
Comment (SX-9) [inline domains/it.json:10]: txt record can only be a string or array of string

**Review by satr14washere — CHANGES_REQUESTED** [review]

File: domains/it.json (hunk 1 above)
Comment (tr1xem) [reply domains/it.json:10]: But it is a github pages record? how do i fix it any solutions?

Comment (SX-9) [reply domains/it.json:10]: GitHub Pages use CNAME not TXT

Comment (tr1xem) [reply domains/it.json:10]: Using it for whole account not a single repo

**Review by satr14washere — APPROVED** [review]
all good

now wait for a maintainer

**Pull request closed without merging**